README.md

```diff
@@ -145,6 +146,45 @@ The optional `revision` value is printed on the title page and in the header, if
 
 The manufacturer name, which must be specified in `system.yml` data document, is also show on the title page.
 
+## Audit Checklists
+
+We include several checklists for various standards. 
+These are used by the `rdm gap` command to check output documents for appropriate references to a given standard. 
```

Comment:
What are you opinions about hard-wrapping vs not hard-wrapping lines in markdown files? (I usually don't hard-wrap lines, mainly so I don't have to maintain the wraps when I edit sentences)

Reply:
I prefer hard wrap for markdown since the lines get joined in the final pretty view anyway. Some view/edit/diff tools require the user to horizontally scroll quite a long distance to see more of the line. This assumes the scroll bar even gets noticed. A diff tool can end up including the whole long paragraph for a small edit. The `grep` like tools also include way too much surrounding context on a search. There are also more possible merge problems if two people both make small edits to different parts of a long paragraph.

Reply:
You make several convincing arguments, several of which I had not heard. I think I'm sold!

Reply:
I may need to update my vim set up to help make it auto hard-wrap